owners/src/local_repo.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2016 Google Inc.
```

Comment:
All files should have "The AMP HTML Authors" instead of "Google Inc." - you don't have to replace all of those in this PR, but do it at least for any new file you're adding

Reply:
Done.